zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -748,6 +749,16 @@ private void removeAllAngularObjectInParagraph(String user, String paragraphId)
   }
 
   public void persist(AuthenticationInfo subject) throws IOException {
```

Comment:
Who will call the `persist` method? and what the meaning of the prefix `force` in front of the `forcePersist`?